### PR TITLE
Mobile/fix depth of field on adreno3 xx

### PR DESCRIPTION
**Pull request by rjonaitis**
---
### Purpose of this PR
Fix case: https://fogbugz.unity3d.com/f/cases/1327076/
The depth of field shader has `pragma target 3.5`. Devices with Adreno 3XX gpus report SystemInfo.graphicsShaderLevel is 35, but due to buggy drivers we have disabled instancing support on them. That causes the depth of field check for SystemInfo.graphicsShaderLevel to pass, but loading the shader fails due to lack of instancing support.

---
### Testing status
Manual testing on Nexus5/Adreno330

---
### Comments to reviewers
Notes for the reviewers you have assigned.

### CODE REVIEW

**Review by simon-engelbrecht-soerensen — APPROVED** [review]

File: com.unity.render-pipelines.universal/Runtime/Passes/PostProcessPass.cs [hunk 1]
```diff
@@ -366,7 +366,10 @@ int GetDestination()
             }
 
             // Depth of Field
-            if (m_DepthOfField.IsActive() && !isSceneViewCamera)
+            // Adreno 3xx SystemInfo.graphicsShaderLevel is 35, but instancing support is disabled due to buggy drivers.
+            // DOF shader uses #pragma target 3.5 which adds requirement for instancing support, thus marking the shader unsupported on those devices.
+            var dofMaterial = m_DepthOfField.mode.value == DepthOfFieldMode.Gaussian ? m_Materials.gaussianDepthOfField : m_Materials.bokehDepthOfField;
```
Comment (phi-lira) [inline com.unity.render-pipelines.universal/Runtime/Passes/PostProcessPass.cs:371]: I dont' understand how the material returns null for Adreno 3xx here.

**Review by phi-lira — APPROVED** [review]
I don't the fix but if this fixes the issue LGTM.

File: com.unity.render-pipelines.universal/Runtime/Passes/PostProcessPass.cs (hunk 1 above)
Comment (rjonaitis) [reply com.unity.render-pipelines.universal/Runtime/Passes/PostProcessPass.cs:371]: If the shader is not supported, it returns null. That's a result of an older identical issue fix https://github.com/Unity-Technologies/Graphics/commit/9157ee7b686785a2b2e0e1cd903ea681fe65b83f

**Review by evelinastaniulyte — APPROVED** [review]
The bug: https://fogbugz.unity3d.com/f/cases/1327076/ is verified as fixed. Depth of field in not initialised on Adreno 3xx devices. 
Tested Adreno 3xx devices:
- VLNQA00104, Lge Nexus 5 (Nexus 5), Android 6.0.1, CPU: Snapdragon 800 MSM8974, GPU: Adreno (TM) 330
- VLNQA00111, Samsung Galaxy J5(2016) (SM-J510FN), Android 7.1.1, CPU: Snapdragon 410 MSM8916, GPU: Adreno (TM) 306 

Also checked some other devices additionally. The Depth of Field is initialised as expected:
- VLNQA00340, Samsung Galaxy Note20 Ultra 5G (SM-N986B), Android 10, CPU: Exynos 990, GPU: Mali-G77
- VLNQA00280, Samsung Galaxy S10+ USA, CPU: Snapdragon 855 SM8150, GPU: Adreno 650, Android: 9
- VLNQA00287, Motorola moto g(7) power (moto g(7) power), Android 9, CPU: Snapdragon 625 MSM8953, GPU: Adreno (TM) 506

**Pull request merged**